src/CliLoggableInterface.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+namespace Olcs\Logging;
+
+interface CliLoggableInterface {
+}
```

Comment:
Would be good to define some methods for static analysis, `getScriptName` & `getParams` at least.

Can use: https://github.com/laminas/laminas-console/blob/master/src/Request.php as a reference of how Laminas did this originally - we could do with something similar.

Reply:
Added thanks